test/blackbox-tests/test-cases/watching/watching-eager-concurrent-build-command.t

```diff
@@ -24,12 +24,20 @@ Build the project once before starting the watch server so the watch server star
 
 Demonstrate that we can run "dune build" while the watch server is running.
   $ dune build
+  Warning: A running dune (pid: 124305) instance has locked the build
+  directory. If this is not the case, please delete "_build/.lock".
+  The requested targets will still be built however functionality will be
+  reduced and some command line arguments will be ignored.
```

Comment:
A couple of questions:

- "If this is not the case, please delete ..." is this relevant ? if the build is being forwarded to a running instance, it doesn't make much sense to me to ask the user to delete the lock file.
- "functionality will be reduced" what does this refer to?

I suggest tweaking the message and making it a bit less scary:

> Warning: your build request is being forwarded to a running Dune instance (pid: 124305). Note that certain command line arguments may be ignored.